lookup/degree_fields.yml

```diff
@@ -0,0 +1,631 @@
+- from: Aeronautical Engineering
+  to: http://opaquenamespace.org/ns/osuDegreeFields/h17mdAyY
+- from: Agricultural Bacteriology
+  to: http://opaquenamespace.org/ns/osuDegreeFields/oFEEwUkk
+- from: Agricultural Chemistry
+  to: http://opaquenamespace.org/ns/osuDegreeFields/2j9jzFJJ
+- from: Agricultural Economics
+  to: http://opaquenamespace.org/ns/osuDegreeFields/EyJ4npVX
+- from: Agricultural Economics and Dairy Manufacturing
+  to: http://opaquenamespace.org/ns/osuDegreeFields/4JZUDEem
+- from: Agricultural Education
+  to: http://opaquenamespace.org/ns/osuDegreeFields/Vl3rhaU2
+- from: Agricultural Engineering
+  to: http://opaquenamespace.org/ns/osuDegreeFields/6jJMIG5D
+- from: Agricultural and Resource Economics
+  to: http://opaquenamespace.org/ns/osuDegreeFields/xVaavcnH
+- from: Agricultural and Resource Economics and Geography
+  to: http://opaquenamespace.org/ns/osuDegreeFields/5lxcWCm2
+- from: Agriculture
+  to: http://opaquenamespace.org/ns/osuDegreeFields/SI1eCX0d
+- from: Agronomic Crop Science
+  to: http://opaquenamespace.org/ns/osuDegreeFields/0ujjmW9P
+- from: Agronomy
+  to: http://opaquenamespace.org/ns/osuDegreeFields/wtcwVchw
+- from: Animal Breeding
+  to: http://opaquenamespace.org/ns/osuDegreeFields/KWzvXUyz
+- from: Animal Husbandry
+  to: http://opaquenamespace.org/ns/osuDegreeFields/OhWpZ9fL
+- from: Animal Industries
+  to: http://opaquenamespace.org/ns/osuDegreeFields/L5df5cGz
+- from: Animal Nutrition
+  to: http://opaquenamespace.org/ns/osuDegreeFields/urkrkTUy
+- from: Animal Science (Nutrition)
+  to: http://opaquenamespace.org/ns/osuDegreeFields/4QGjGXDe
+- from: Animal Science
+  to: http://opaquenamespace.org/ns/osuDegreeFields/ZfhfkCsy
+- from: Animal Sciences
+  to: http://opaquenamespace.org/ns/osuDegreeFields/6UkQ0fd4
+- from: Anthropology
+  to: http://opaquenamespace.org/ns/osuDegreeFields/uWcU7XNs
+- from: Apparel, Interiors, Housing, and Merchandising
+  to: http://opaquenamespace.org/ns/osuDegreeFields/aAynzoLm
+- from: Apparel, Interiors, and Merchandising
+  to: http://opaquenamespace.org/ns/osuDegreeFields/O5sXIL5K
+- from: Applied Anthropology
+  to: http://opaquenamespace.org/ns/osuDegreeFields/iPVxZthZ
+- from: Applied Economics
+  to: http://opaquenamespace.org/ns/osuDegreeFields/LadD13bk
+- from: Applied Ethics
+  to: http://opaquenamespace.org/ns/osuDegreeFields/anT08Cgv
+- from: Applied Mathematics
+  to: http://opaquenamespace.org/ns/osuDegreeFields/m0dVUKke
+- from: Applied Physics
+  to: http://opaquenamespace.org/ns/osuDegreeFields/kfG76Jz2
+- from: Atmospheric Science
+  to: http://opaquenamespace.org/ns/osuDegreeFields/wjumKmKM
+- from: Atmospheric Sciences
+  to: http://opaquenamespace.org/ns/osuDegreeFields/XRUAakIE
+- from: Automotive Engineering
+  to: http://opaquenamespace.org/ns/osuDegreeFields/T2NgrYg1
+- from: Bacteriology
+  to: http://opaquenamespace.org/ns/osuDegreeFields/on81oQFO
+- from: Biochemistry &amp; Biophysics
```

Comment:
Will this `&amp;` work, is this what is in the metadata or does it have `&` instead?